core/src/test/scala/unit/kafka/server/ISRExpirationTest.scala

```diff
@@ -165,6 +178,13 @@ class IsrExpirationTest {
     allReplicas.foreach(r => partition.addReplicaIfNotExists(r))
     // set in sync replicas for this partition to all the assigned replicas
     partition.inSyncReplicas = allReplicas.toSet
+    // set lastCatchUpTime to current time
+    for(replica <- partition.assignedReplicas() - leaderReplica)
+      replica.updateLogReadResult(new LogReadResult(FetchDataInfo(new LogOffsetMetadata(0L), MemoryRecords.EMPTY),
+                                  hw = 0L,
+                                  leaderLogEndOffset = 0L,
+                                  fetchTimeMs =time.milliseconds,
+                                  readSize = -1))
```

Comment:
Ok, thanks for the explanation. Could you add a space after = in =time.milliseconds?